frontend/src/components/Projectile.tsx

```diff
@@ -0,0 +1,68 @@
+import React, { useEffect, useState } from "react";
+
+interface Props {
+  source: string;
+  target: string;
+  duration: number;
+  positiveNews: boolean;
+}
+
+interface Coordinates {
+  x: number;
+  y: number;
+}
+
+const center = (element: HTMLElement | null): Coordinates => {
+  if (!element) return { x: 0, y: 0 };
```

Comment:
Hmm...
I just experienced a situation in which the battle has finished, the board has disappeared, but the lasers were still flying, and then one of them got shot into space (0, 0), because it could not find its target anymore. Probably can be solved by changing the timing I wrote about above.

Reply:
yeah, the timing is messed up now